cwl/src/main/scala/cwl/Workflow.scala

```diff
@@ -30,21 +30,24 @@ case class Workflow private(
                      steps: Array[WorkflowStep],
                      requirements: Option[Array[Requirement]],
                      hints: Option[Array[Hint]]) {
+
+  steps.foreach { _.parentWorkflow = this }
+
   /** Builds an `Executable` from a `Workflow` CWL with no parent `Workflow` */
   def womExecutable(validator: RequirementsValidator, inputFile: Option[String] = None): Checked[Executable] = {
     CwlExecutableValidation.buildWomExecutable(womDefinition(validator), inputFile)
   }
 
-  private[cwl] var parentWorkflow: Option[Workflow] = None
+  private[cwl] var parentWorkflowStep: Option[WorkflowStep] = None
```

Comment:
ToL: This ugly var doesn't get the comment that the other two do

Reply:
oops